hrsi_representation/scripts/online_qtc_creator.py

```diff
@@ -115,6 +127,9 @@ def ppl_callback(self, msg):
         }
         self._msg_buffer.append(msgs)
 
+    def poseSt_callback(self, msg):
+        self.pose_callback(msg.pose)
```

Comment:
What is the point of this?

Reply:
I didn't have a Pose publisher but a PoseStamped publisher for the robot pose. It seemed easier than finding out how to make robot to publish a Pose instead of Pose stamped.